.github/actions/configure-aws-credentials/action.yml

```diff
@@ -0,0 +1,80 @@
+name: "Configure AWS credentials"
+
+description: "Configures credentials for the selected AWS account"
+
+inputs:
+  environment_name:
```

Comment:
Is this really the account you want to use, rather than environment name?

Reply:
And maybe make the default `tools`?

Reply:
Default is set to `tools`